tests/runtests.py

```diff
@@ -243,7 +243,7 @@ def actual_test_processes(parallel):
 
 
 def django_tests(verbosity, interactive, failfast, keepdb, reverse,
-                 test_labels, debug_sql, parallel):
+                 test_labels, debug_sql, parallel, tags=None, exclude_tags=None):
```

Comment:
It's been a while since I reviewed this, but I think what I meant was, why can't they be args?